pages/linux/e2label.md

```diff
@@ -0,0 +1,7 @@
+# e2label
+
+> Change the label on an ext2/ext3/ext4 filesystem.
+
+- Change the volume label on the ext2/ext3/ext4 filesystem:
```

Comment:
Perhaps this will read better?
```suggestion
- Change the volume label on a specific partition:
```

Reply:
I would still change this to "a specific ext partition" or perhaps "a specific ext-type partition".

Reply:
@zdroid done.